no-computer.md

```diff
@@ -1,44 +1,52 @@
-# No computer project
-
+# Violin
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Violin is a hard instrument. Rather than having a separated fingerbaord for each note, like frets on a guitar, violin simply has a continuous fingerboard. The violin produces different notes based off of where fingers are placed and in what combination. Having a language that emulates what happens when some combination of fingers are placed would allow users to focus more on the quality of the music rather than on producing the right sound. My computer-free project would be for novices learning violin for the first time, or people who don't have time to formally learn violin but have composition ideas. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language would eliminate the need to practice (for years usually) to train both the ear to identify notes and the fingers to produce the correct notes. This language would help beginners understand proper finger placement as well as amateur composers that simply want to create a violin piece. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This "DSL" is appropriate because all violin players use the same "DSL". Therefore, when players play the same piece, they are similar in sound. However, this "DSL" also has flexibility in that there are many ways to play the same piece. Though the notes may be exactly the same, the "DSL" allows the user to express their personal preference and tastes which is reflected in their sound.
 
```

Comment:
It's interesting to think of music (and this specific interface for it) as a DSL. Each of the notes are tools with which songs, or programs, can be constructed. I like this general idea as well, since songs are programs and playing them is equal to running the program. Errors are easy to hear and there's instant feedback from the language if the user "throws an error," or plays the wrong note.